### ISSUE TITLE
SRA Toolkit downloads failing with segmentation faults

### ISSUE THREAD

**Issue by ryanpe13002** [ncbi/sra-tools#745]
To whom it may concern,

I am working on downloading Fastq files from SRA and I am getting a very strange set of errors that look like they are due to a segmentation fault. This error is repeated and recurrent; no matter how many times I try to rerun the script, it gives me the same output.

Thanks so much for any help or guide you can provide on how I can resolve this error!

My download script (for Linux-based HPC, operated under Sumner): [0_test_download.sh.zip](https://github.com/ncbi/sra-tools/files/10147438/0_test_download.sh.zip)

My error file: 
[SRA_DL_16753261_4294967294.err.zip](https://github.com/ncbi/sra-tools/files/10147441/SRA_DL_16753261_4294967294.err.zip)

Yours most sincerely,
Ryan Englander

MD/PhD candidate, Jackson Laboratory for Genomic Medicine

**Comment by wraetz** [comment]
The commands in your test-download-script are correct. Unfortunately we are not familiar with Slurm/Sumner. To better understand the problem - can you run just the 'fasterq-dump' command on a stand-alone Linux-machine. ( since you already have downloaded the SRA-accession 'SRR655447' and it's dependencies. ). By the way '-e 6' can be omitted, it is the default.

**Comment by wraetz** [comment]
Just in case there are differences in the files involved, please verify against these md5-sums:

$md5sum ./SRR655447/*
2d81051cbf2805983b070bce6bc533a1  ./SRR655447/GL000196.1
91a405504997f735f034ac9d0a7eb1c2  ./SRR655447/GL000198.1
3b83c8787963c075415122e55ccba182  ./SRR655447/GL000201.1
a346c9b46a754baf2839ccbc1724ba37  ./SRR655447/GL000202.1
f8dd8dc3ad58f22236c0c32a5ad8c22b  ./SRR655447/GL000203.1
d0652dfa8d70a036b65f835e7b4a1922  ./SRR655447/GL000204.1
f5bc43b7786feb4d66efacf4be8d64bc  ./SRR655447/GL000206.1
b2ca7227161676195a749733acfe82dc  ./SRR655447/GL000207.1
3559c4bfd4260c555f1966e141d87b55  ./SRR655447/GL000209.1
5ac6937435410cde3c56f7b2054d4743  ./SRR655447/GL000210.1
453d86ca645a5cf2c47837bb30e14da6  ./SRR655447/GL000213.1
f9f652c7f3d12266c5baa062b1cf5d98  ./SRR655447/GL000215.1
97b95fe1ac39637ed1a3c68159db65e3  ./SRR655447/GL000227.1
3ebcb41ca75e414dae5ec9c88e300f63  ./SRR655447/GL000228.1
0b988a48c2599a606fb03a95f9bc3802  ./SRR655447/GL000232.1
a8481b22acd771d64077a22d4d28b4b2  ./SRR655447/GL000233.1
e483626c20661252291dbe28cc430f2a  ./SRR655447/GL000236.1
a6fadcb133d57b3ef187af69170f3284  ./SRR655447/GL000237.1
13b48448d82e955d93f25f792cc662ff  ./SRR655447/GL000238.1
fb69cb7dc9c1c3f67111c7ecddeb12b8  ./SRR655447/GL000240.1
b90c24645ecde4d7acd59674b00583b8  ./SRR655447/GL000242.1
c3daa6dd6de0c1117ebb995173e0ae1b  ./SRR655447/GL000243.1
30eb6114adf9bf25e2c8e04bb933c20c  ./SRR655447/GL000247.1
f013f4e4bcd8d2bb9e002880586f9264  ./SRR655447/GL000248.1
4cc765b94d4c7bbbf6a8603dfa292b61  ./SRR655447/GL000249.1
dbe58034e1ae1792cbdca810407a3672  ./SRR655447/SRR655447.sra
f08594688c975459b44bb8f90821b57b  ./SRR655447/SRR655447.sra.vdbcache

md5sum prj_12900_D30277.ngc
c774fb19eb6fdacd6c66eb835eb4b6f9  prj_12900_D30277.ngc

**Comment by ryanpe13002** [comment]
Hi,

So I did some playing around with the system using commands directly on my HPC (which operates off Linux), and it looks like I was able to find a solution that works.

Oddly, the prefetch commands looks like it doesn't produce any output unless an output directory is explicitly specified. When I changed the command I used in my script to the following: 

/home/englar/sratoolkit.3.0.1-centos_linux64/bin/prefetch-orig.3.0.1 SRR655447 -O /sdata/dbGaP/12900/englar/test_GTEx/output --force yes --ngc /home/englar/KEYS/prj_12900_D30277.ngc --max-size 100G

it successfully produced an output. I then changed the input in the fastq-dump command from the SRA accession to the relative path pointing to the downloaded *.sra file, and that seemed to work as well:

/home/englar/sratoolkit.3.0.1-centos_linux64/bin/fastq-dump-orig.3.0.1 output/SRR655447/SRR655447.sra --split-files -O /sdata/dbGaP/12900/englar/test_GTEx/output --ngc /home/englar/KEYS/prj_12900_D30277.ngc

I did verify that the md5sum's you sent me are the same as they appear in the full filelist from the download. However, there are additional files as well; I have attached the full filelist with md5sums here: 
[md5sums.txt](https://github.com/ncbi/sra-tools/files/10166555/md5sums.txt)

So, it looks like it's working!
